docs/DeveloperGuide.md

```diff
@@ -357,6 +366,14 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding a Client
+
+1. Adding a Client while all Clients are being shown
+
+   1. Test case: `cadd n/David …​` <br>
+      Expected: First contact is added to the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
I'm not too sure if there is a timestamp in the status bar 😅 , but it is also in the AB3 DG ...

Reply:
hmm but for now, we are still largely following AB3 so it should still be there?